cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Gov/CommitteeAddNew.hs

```diff
@@ -120,12 +120,51 @@ hprop_constitutional_committee_add_new = integrationWorkspace "constitutional-co
       ccColdVKeyFp n = gov </> "cc-" <> show n <> "-cold.vkey"
       stakeVkeyFp = gov </> "stake.vkey"
       stakeSKeyFp = gov </> "stake.skey"
+      stakeCertFp = gov </> "stake.regcert"
 
   cliStakeAddressKeyGen
     $ KeyPair { verificationKey = File stakeVkeyFp
               , signingKey = File stakeSKeyFp
               }
 
+  -- Register stake address
+
+  void $ execCli' execConfig
+    [ eraName, "stake-address", "registration-certificate"
```

Comment:
We should abstract this out as a standalone function (not in this PR). cc: @carbolymer